.github/workflows/swagger_docs.yml

```
@@ -0,0 +1,75 @@
+name: Generate Endpoint docs and swagger.yaml
```

```suggestion
name: Generate Endpoint Docs
```

Ig a short name is enough.